src/types/connection/page_info.rs

```diff
@@ -1,20 +1,17 @@
-#![allow(missing_docs)]
-
-use crate::types::connection::cursor::Cursor;
 use async_graphql_derive::SimpleObject;
 
 /// Information about pagination in a connection
-#[SimpleObject(internal, desc = "Information about pagination in a connection.")]
+#[SimpleObject(internal)]
 pub struct PageInfo {
-    #[field(desc = "When paginating backwards, are there more items?")]
+    /// When paginating backwards, are there more items?
     pub has_previous_page: bool,
 
-    #[field(desc = "When paginating forwards, are there more items?")]
+    /// When paginating forwards, are there more items?
     pub has_next_page: bool,
 
-    #[field(desc = "When paginating backwards, the cursor to continue.")]
-    pub start_cursor: Option<Cursor>,
+    /// When paginating backwards, the cursor to continue.
+    pub start_cursor: Option<String>,
```

Comment:
`Option<impl CursorType>`?

Reply:
This requires `PageInfo<C>`, but it's an internal type, so `String` might be more convenient.